File: app/src/main/java/restaurant/sa/com/sarestaurant/appview/weather/model/Location.java

package restaurant.sa.com.sarestaurant.appview.weather.model;
import com.google.gson.annotations.SerializedName;
public class Location {

    @SerializedName("city")
    private String mCity;
    @SerializedName("country")
    private String mCountry;
    @SerializedName("region")
    private String mRegion;

    public String getCity() {
        return mCity;
    }

    public void setCity(String city) {
        mCity = city;
    }

    public String getCountry() {
        return mCountry;
    }

    public void setCountry(String country) {
        mCountry = country;
    }

    public String getRegion() {
        return mRegion;
    }

    public void setRegion(String region) {
        mRegion = region;
    }

}

File: app/src/main/java/restaurant/sa/com/sarestaurant/appview/weather/model/Location.kt
package restaurant.sa.com.sarestaurant.appview.weather.model

import com.google.gson.annotations.SerializedName

class Location {

    @SerializedName("city")
    var city: String? = null
    @SerializedName("country")
    var country: String? = null
    @SerializedName("region")
    var region: String? = null

}
